verticapy/tests/vModel/test_model_selection.py

```diff
@@ -22,8 +22,8 @@
 import matplotlib.pyplot as plt
 
 # VerticaPy
-import verticapy
-from verticapy.core.vdataframe.vdataframe import vDataFrame
+import verticapy, drop
```

Comment:
shouldnt it be from verticapy import drop?

Reply:
correct